src/Hedgehog/Gen.fs

```diff
@@ -551,3 +547,13 @@ module GenBuilder =
 module GenOperators =
     let (<!>) = Gen.map
     let (<*>) = Gen.apply
+
+open System
+
+/// In C# friendly manner, a generator for values and shrink trees of type 'a
+type Gen =
+    static member Bool = Gen.bool
```

Comment:
The issue here is only for those generators taking `FSharpFunc` for example `Gen.delay`. I don't think we need to carry on each and every generator.